spec/controllers/conferences_controller_spec.rb

```diff
@@ -23,6 +23,15 @@
         get :show, id: conference.short_title
         expect(response).to render_template :show
       end
+
+      it 'assigns correct conference from a custom domain' do
```

Comment:
This `it` block should be within a context:

`context "accessing via a custom domain"` or something like that.

Then the conference's attribute can be set with a let:
`let(:conference) { create(:conference, custom_domain: 'thebest.conf', splashpage: create(:splashpage, public: true), venue: create(:venue)) }`

And `@request` can be set in a `before` block:

`before { @request.host = 'thebest.conf' }`

So you can reduce your test to three lines :)